src/arguments.js

```diff
@@ -54,7 +54,24 @@ const getDefaultValue = (description, regexp) => {
  */
 const getEnum = (string, context) => {
     const result = getValueByRegexp(string, context.regexp.enumValues.enum);
-    return result ? result[1].split(context.regexp.enumValues.split) : null;
+    return result ? removeExtraEnumValues(
+        result[2].split(context.regexp.enumValues.split), context) : null;
```

Comment:
Why change from 1 to 2 index of result?
Also I propose to create const for `split` function.

Reply:
@svoboda-rabstvo because regexp for enum have changed